modules/ROOT/partials/menu-item-ids/footnotes-menu-items.adoc

```diff
@@ -0,0 +1,6 @@
+[cols="1,1,2",options="header"]
+|===
+|Menu item identifier |xref:menus-configuration-options.adoc#example-the-tinymce-default-menu-items[Default Menu Location] |Description
+|`+footnotes+` |Insert |Inserts a superscripted footnote number into the editor at the selection, and creates a new entry in the footnotes section linked to that footnote.
+|`+footnotesupdate+` |Insert |Manually updates the footnotes section.
```

Comment:
```suggestion
|`+footnotesupdate+` |_Not Applicable_ |Manually updates the footnotes section.
```

Reply:
_Not Applicable_ here reads as to say `footnotesupdate` should not have an entry in `footnotes-menu-items.adoc`.

If it doesn’t ever appear in a menu, it doesn’t need to listed here.

Am I inferring from the text correctly here?

Reply:
Turned my questions into a ticket: [DOC-1805](https://ephocks.atlassian.net/browse/DOC-1805).